File: Heat/Models/Boiler.vb
Namespace Models
    ''' <summary>
    ''' Rappresenta una caldaia
    ''' </summary>
    ''' <remarks></remarks>
    Public Class Boiler
        Property ID As Integer
        Property Manifacturer As Manifacturer
        Property Model As ManifacturerModel
        Property SerialNumber As String
        Property InstallationDate As DateTime
        Property FirstStartUp As DateTime
        Property Warranty As String
        Property WarrantyExpiration As DateTime

        ''' <summary>
        ''' Elenco delle operazioni di manutenzione eseguite sulla caldaia. 
        ''' </summary>
        ''' <value></value>
        ''' <returns></returns>
        ''' <remarks></remarks>
        Property Services As List(Of BoilerService)

    End Class
End Namespace

File: Heat.ConvertedToC#/Models/Boiler.cs
using System;
using System.Collections.Generic;
namespace Heat.Models
{
    /// <summary>
    /// Rappresenta una caldaia
    /// </summary>
    /// <remarks></remarks>
    public class Boiler
	{
		public int ID { get; set; }
		public Manifacturer Manifacturer { get; set; }
		public ManifacturerModel Model { get; set; }
		public string SerialNumber { get; set; }
		public DateTime InstallationDate { get; set; }
		public DateTime FirstStartUp { get; set; }
		public string Warranty { get; set; }
		public DateTime WarrantyExpiration { get; set; }

		/// <summary>
		/// Elenco delle operazioni di manutenzione eseguite sulla caldaia. 
		/// </summary>
		/// <value></value>
		/// <returns></returns>
		/// <remarks></remarks>
		public List<BoilerService> Services { get; set; }

	}
}
